cmd/api/src/api/saml/saml.go

```diff
@@ -348,7 +349,13 @@ func (s ProviderResource) serveAssertionConsumerService(response http.ResponseWr
 			log.Errorf("[SAML] Failed to lookup user for SAML provider %s: %v", s.serviceProvider.Config.Name, err)
 			s.writeAPIErrorResponse(request, response, http.StatusBadRequest, "session assertion does not meet the requirements for user lookup")
 		} else {
-			s.authenticator.CreateSSOSession(request, response, principalName, s.serviceProvider.Config)
+			s.authenticator.CreateSSOSession(request, response, principalName, model.SSOProvider{
+				Type: model.SessionAuthProviderSAML,
+				Name: s.serviceProvider.Config.Name,
+				Slug: s.serviceProvider.Config.Name,
```

Comment:
It doesn't _look_ like this gets used, but, there is the potential that the `Slug` would be different than the `Name` in the database.
It looks like it should be fine here,  but wanted to call that out.